libcob/fileio.h

```diff
@@ -47,10 +47,14 @@
 #include <sys/stat.h>
 #include <limits.h>
 
+#if defined (COB_EXPERIMENTAL) && defined (HAVE_LOCALE_H)
+#include <locale.h>
+#endif
+
```

Comment:
as this is only to be used in fileio.c - please move that include there

Reply:
Done.